cmd/environment.go

```diff
@@ -130,7 +130,7 @@ func newEnvironmentCreateCmd() *cobra.Command {
 	flags.StringVarP(&consensus, "consensus", "k", "raft", "consensus algorithm to use for the given protocol, raft or ibft for quorum, poa for geth")
 	flags.BoolVarP(&multiRegion, "multi-region", "R", false, "whether to enable multi region")
 	flags.IntVarP(&blockPeriod, "block-period", "P", 0, "block period in seconds")
-	flags.UintVarP(&chainID, "chain-id", "c", 0, "Chain ID")
+	flags.UintVarP(&chainID, "chain-id", "C", 0, "Chain ID")
```

Comment:
Nice catch! I'm assuming there is going to be a follow-on update to the Terraform provider?

Reply:
Thanks! What would need to be updated in the Terraform provider?